src/components/protocol_handler/test/protocol_handler_tm_test.cc

```diff
@@ -931,6 +931,9 @@ TEST_F(ProtocolHandlerImplTest, FloodVerification) {
                   message_id,
                   &some_data[0]);
   }
+#ifdef OS_WIN32
+  Sleep(1000);
```

Comment:
Here add below add many ```Sleep()``` for  OS_WIN32 port, it's just for Win32 port? Why add this timer delay? if necessary, please add comments for other developers.